submissions/dummy_submission/the-nuclear-options-v1.0.md

```diff
@@ -0,0 +1,233 @@
+
+* **Nuclear calculations and designs are hard **– At the end of the day, calculating ratios for nuclear setups is not that different from planning your assembly lines. In the same spirit, the wonderful [Factorio Cheat Sheet](https://factoriocheatsheet.com/#nuclear-power) provides the formulas and interactive calculators you need and there are just a few design rules to keep in mind, which we cover below.
+* **Nuclear plants are bad for the game performance (UPS)** – It’s true that with all the fluid mechanics involved, nuclear cannot compete with the simplicity of solar power in terms of UPS efficiency, but that does not mean its impact is that large, especially with just a handful of reactors.  We will cover this too, below.
+* **What if the radiation causes mutated mega-biters?** - Oddly enough radiation does not seem to affect any entities in vanilla Factorio while regular old pollution already is enough to mutate those pesky biters. Nuclear energy is actually very helpful in this regard, as creates no pollution apart from the relatively small amounts from uranium mining and processing.
+
+With these concerns addressed, let us look into the DIY of nuclear power.
+
+## Designing your own plants
+
+### How do I know how many components I need?
+
+When starting out, it makes sense to start with single reactor, or perhaps a 1x2 (1 row of 2 reactors) layout. The [wiki tutorial](https://wiki.factorio.com/Tutorial:Nuclear_power) has you covered for the basics. After that, the size of your design is based on your power needs. You can arrange your reactors however you want, and the easy way to calculate the power output from a layout is making a single reactor equivalent (SRE) diagram. It works in three steps. We can go over the steps with an example of a 3x3 setup:
+
+`1) Draw a copy of your reactor arrangement and to each reactor assign a value of 1:`
+
+``[1][1][1]` 
+
+``[1][1][1]` 
+
+``[1][1][1]` 
+
+`2) For each reactor, count the number of neighbors it has (they must be aligned exactly side by side) and add it to the 1:`
+
+``[1+2][1+3][1+2] -> [3][4][3]` 
+
+``[1+3][1+4][1+3] -> [4][5][4]` 
+
+``[1+2][1+3][1+2] -> [3][4][3]` 
+
+`3) Sum all the numbers and multiply by 40 to get your power output in MW:`
+
+`(3 + 4 + 3) * 2 + (4 + 5 + 4) = 33 SRE`
+
+`33 SRE * 40MW = 1320MW`
+
+Next to harvest all that reactors' heat power we need heat exchangers. To find how many you need, simply divide the total power produced by 10, since one exchanger consumes 10MW. For our example we get 132. Meanwhile, to feed these heat exchangers with water, 1 offshore pump is enough for 12 heat exchangers, hence in this example we need 132 / 12 = 11 offshore pumps.

```

Comment:
Yeah, I agree with Firerazor. I think you need to say something like:

We know our nuclear reactors can generate 1320MW of heat, but now the heat needs to be turned into steam. A single heat exchanger can exchange 10MW of heat, so in order to turn all heat to steam we will need 1320 / 10 = 132 nuclear reactors.

According to the wiki the heat exhanger can output ~103 steam per second, meaning it needs about 103 water per second. Seeing as offshore pump can produce 1200 water per second a single pump can support 1200 / 103 = 11 heat exchangers.

Reply:
Precisely that, yes. I saw that I could not avoid explaining these ratios, so might as well spell them out. I'll do the same with steam turbines (consumes 60/s vs 103.09/s produced by exchangers).